requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest
+flake8
```

Comment:
frankly speaking that's already in our travis repo as dedicated step, we may skip `flake8` in our dependencies for test framework